examples/all-clusters-app/ameba/main/include/ManualDishWasherAlarmCommand.h

```diff
@@ -0,0 +1,129 @@
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "controller/InvokeInteraction.h"
+#include "controller/ReadInteraction.h"
+#include <app/clusters/dishwasher-alarm-server/dishwasher-alarm-server.h>
+
+#if CONFIG_ENABLE_CHIP_SHELL
+#include "lib/shell/Engine.h"
+#include "lib/shell/commands/Help.h"
+#endif // ENABLE_CHIP_SHELL
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::DishwasherAlarm;
+
+#if CONFIG_ENABLE_CHIP_SHELL
+using Shell::Engine;
+using Shell::shell_command_t;
+using Shell::streamer_get;
+using Shell::streamer_printf;
+
+Engine sShellManualDishWasherAlarmStateSubCommands;
+#endif // defined(ENABLE_CHIP_SHELL)
+
+#if CONFIG_ENABLE_CHIP_SHELL
+
+CHIP_ERROR ManualDishWasherAlarmCommandHelpHandler(int argc, char ** argv)
+{
+    sShellManualDishWasherAlarmStateSubCommands.ForEachCommand(Shell::PrintCommandHelp, nullptr);
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ManualDishWasherAlarmCommandHandler(int argc, char ** argv)
+{
+    if (argc == 0)
+    {
+        return ManualDishWasherAlarmCommandHelpHandler(argc, argv);
+    }
+
+    return sShellManualDishWasherAlarmStateSubCommands.ExecCommand(argc, argv);
+}
+
+CHIP_ERROR ManualDishWasherAlarmSetRaiseCommandHandler(int argc, char ** argv)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    EmberAfStatus status;
+    DishwasherAlarmServer & serverInstance = DishwasherAlarmServer::Instance();
+
+    status = serverInstance.SetSupportedValue(1, 47);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        err = CHIP_ERROR_INTERNAL;
+        goto exit;
+    }
+
+    status = serverInstance.SetMaskValue(1, 47);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        err = CHIP_ERROR_INTERNAL;
+        goto exit;
+    }
+
+    status = serverInstance.SetStateValue(1, 14, 0);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        err = CHIP_ERROR_INTERNAL;
+        goto exit;
+    }
+exit:
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(DeviceLayer, "ManualDishWasherAlarmSetRaiseCommandHandler Failed!\r\n");
+    }
+
+    return err;
+}
+
+CHIP_ERROR ManualDishWasherAlarmSetLowerCommandHandler(int argc, char ** argv)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    EmberAfStatus status;
+    DishwasherAlarmServer & serverInstance = DishwasherAlarmServer::Instance();
+
+    status = serverInstance.SetSupportedValue(1, 47);
```

Comment:
Might be good to document where these magic "47" and "14" values are coming from.  Maybe by writing them as actual bitwise "or" of the relevant bits.

Reply:
Updated to use BitMask<AlarmMap> instead for clearer code.